certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/action/model/ActionRuleChildRelationNotSigned.java

```diff
@@ -0,0 +1,24 @@
+package se.inera.intyg.certificateservice.domain.action.model;
+
+import java.util.List;
+import java.util.Optional;
+import se.inera.intyg.certificateservice.domain.certificate.model.Certificate;
+import se.inera.intyg.certificateservice.domain.certificate.model.Relation;
+import se.inera.intyg.certificateservice.domain.certificate.model.Status;
+
+public class ActionRuleChildRelationNotSigned implements ActionRule {
```

Comment:
We have the rules today called ActionRuleChildRelationNoMatch and ActionRuleChildRelationMatch. 

Maybe it is time to expand them what statuses to consider. Right now they consider certain relations and have hardcoded what statuses are relevant.

Reply:
Added allowedStatuses to ActionRuleChildRelationNoMatch to handle statuses more dynamically.